CMakeLists.txt

```diff
@@ -0,0 +1,157 @@
+# ----------------------------------------------------------------------
+# 2020 Bernd Pfrommer bernd.pfrommer@gmail.com
+#
```

Comment:
Please include our copyright header as in https://github.com/uzh-rpg/vilib/blob/master/visual_lib/Makefile#L1

In the files you edited, if you whish (and take responsibility) you can add your name to the header in a section below the part that says
```
# Copyright (c) 2019-2020 Balazs Nagy,
# Robotics and Perception Group, University of Zurich
```

Reply:
done